base/agda/Base/Prelude.agda

```diff
@@ -1 +1,7 @@
 module Base.Prelude where
+
+open import Base.Free.Bool    public
+open import Base.Free.Integer public
+open import Base.Free.List    public
+open import Base.Free.Pair    public
+open import Base.Free.Unit    public
```

Comment:
I think the modules that are part of the `Prelude` should be moved from `./base/agda/Base/Free` to `./base/agda/Base/Prelude`.